live-examples/css-examples/basic-box-model/overscroll-behavior-x.html

```diff
@@ -0,0 +1,54 @@
+    <section id="default-example" class="default-example">
+        <div class="example-container">
+            <div class="box">
+                This is a scrollable container. Michaelmas term lately over, and the Lord Chancellor sitting in Lincoln's Inn Hall. Implacable November weather. As much mud in the streets as if the waters had but newly retired from the face of the earth.
+                <br><br>
+                Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.
+            </div>
+            <div id="example-element">
+                <div id="element-left">
+                    This is the inner container. Focus on this container, scroll to the right and when you reach the right edge keep scrolling.
+                    <p>If you have <code class="language-css">overscroll-behavior-x: 
+                        auto;</code> selected the outer container 
+                        will start to scroll.</p>
+                    If you have <code class="language-css">overscroll-behavior-x: 
+                        contain;</code> selected, the outer container will not scroll unless you move your cursor out of the inner container and try to perform scroll on the outer container. 
+
+                        **Note that this applies only to the x-axis. You should still be able to scroll past in the y-axis.
+                </div>
+                <div id="element-right">
+                    This is sample text to fill up space. Lorem ipsum dolor sit amet consectetur adipisicing elit. Eius veniam, animi laudantium quibusdam vitae cumque officiis in eos laborum molestiae quasi ex eaque minima architecto, inventore natus fuga tempora accusamus velit deserunt numquam, illo nemo labore accusantium. Earum nihil placeat sequi veritatis tempora, corrupti et necessitatibus quos! Rem, sed quod!
+                </div>
+            </div>
+            <div id="last-element">
+                This is just some sample text. Lorem ipsum dolor, sit amet consectetur adipisicing elit. Eum at aut eius aliquam blanditiis itaque unde reiciendis? Maxime obcaecati possimus dicta quidem nemo consectetur facere officiis assumenda laudantium. Totam vitae nisi pariatur voluptate et architecto officia ullam cum recusandae! Earum, soluta eveniet corrupti quibusdam libero numquam! Incidunt quidem libero, et ratione aspernatur, officia quisquam cum maxime consectetur pariatur tenetur sint inventore doloremque beatae quibusdam ipsam harum minima nulla deserunt ex illum? Obcaecati animi ab reprehenderit esse adipisci, optio vitae incidunt. Reprehenderit corrupti, autem eum laboriosam in cum! Reprehenderit repellendus, eos magnam deleniti, excepturi sequi rem sint fugit corrupti autem quaerat.
```

Comment:
I think most of this text is unnecessary. It's good to allow users to scroll through the content horizontally and vertically, so they can see how CSS property affects both axes, but a single short sentence would be enough to achieve this.